backend/services/theme_service.py

```
@@ -114,9 +114,9 @@ async def generate_with_face_reference(
                 logger.info(f"🔄 EXIF orientation applied, final size: {pil_image.size}")
                 
                 # 🎯 FULL IMAGE APPROACH: No cropping, no masking - rely on balanced IP-Adapter weight
-                # Theory: IP-Adapter weight ~0.7 + CFG 15.0 = face preserved, body/background from prompt
-                logger.info(f"🎯 Using FULL IMAGE approach with balanced IP-Adapter weight (~0.7)")
-                logger.info(f"🎯 Theory: Balanced IP weight preserves face identity while allowing themed transformation")
+                # Theory: IP-Adapter weight 0.3 + CFG 15.0 = face preserved, body/background completely transformed
+                logger.info(f"🎯 Using FULL IMAGE approach with optimal IP-Adapter weight (0.3)")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)